cryptol.cabal

```diff
@@ -63,7 +64,7 @@ library
                        pretty            >= 1.1,
                        process           >= 1.2,
                        random            >= 1.0.1,
-                       sbv               >= 8.1,
+                       sbv               >= 8.7,
```

Comment:
Are we also missing a modification [here](https://github.com/GaloisInc/cryptol/blob/62a014384b3528d869c1310b5c13bcfeaf2cdeef/cryptol.cabal#L287)?

Reply:
Oh, I didn't see that one. *shrug* I usually don't like to put bounds on test/benchmark suites for packages that are already dependencies of the main library because they're redundant (as far as I know).  Probably we should just remove it.

Reply:
That's probably what I'd do, especially given that none of the other packages there have constraints.